src/ios/CDVRadar.m

```diff
@@ -74,6 +78,27 @@ - (void)setUserId:(CDVInvokedUrlCommand *)command {
     [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
 }
 
+- (void)setLogLevel:(CDVInvokedUrlCommand *)command {
+    NSString *level = [command.arguments objectAtIndex:0];
+
+    RadarLogLevel logLevel = RadarLogLevelNone;
+    if (level) {
+        if ([level isEqualToString:@"error"] || [level isEqualToString:@"ERROR"]) {
```

Comment:
FWIW (and I know that it doesn't matter much, because this isn't called often), its more efficient to uppercase the `level` and just do comparisons against uppercase possibilities. 

```
NSString *level = [command.arguments objecAtIndex:0].uppercased;

if ([level isEqualToString:@"LEVEL"]) {
    ...
```